downstream/modules/eda/con-custom-credential-types.adoc

```diff
@@ -0,0 +1,62 @@
+[id="eda-custom-credential-types"]
+
+= Custom credential types
+
+As a system administrator, you can define a custom credential type works in ways similar to existing credential types in a standard format using a YAML or JSON-like definition. For example, a custom credential type can inject an API token for a third-party web service into an environment variable for your rulebook or custom inventory script to consume.
```

Comment:
@jself-sudoku This statement "can inject an API token for a third-party web service into an environment variable for your rulebook or custom inventory script to consume." is not correct. Currently we only support extra_vars, AAP supports extra_var/file/env but EDA currently only supports extra_var. This could be treated as a deficiency in the EDA implementation of Credential Types. We might have to check with Joe P to see if its ok.

Reply:
Sounds like we should remove this sentence altogether. I'll take care of that.